README.md

```diff
@@ -78,18 +78,18 @@ def format_dolly_dataset(example):
 
 
 def main():
-    # 📊 Load instruction-following dataset
```

Comment:
Why? You are another emoji hater? 👎 🙄

Reply:
No I am the one who added them in the first place!

But I think they should not be in the code, just the text around it.